content/en/agent/faq/certificate_verify_failed-error.md

```diff
@@ -98,11 +107,33 @@ Datadog recommends keeping up to date and updating to the latest version of the
 
 Yes. The certificate is just a preset for the client to use and is not necessary to connect via SSL. Datadog Agent endpoints only accept SSL traffic.
 
+### Windows Agent 5.x configured to use a proxy or the curl http client
+
+This section applies to the Windows Agent 5.x (`<= 5.32.6`), if the Agent is configured to either:
+
+* use a proxy with the `proxy_host` configuration option in `datadog.conf` or the `HTTPS_PROXY` environment variable, or
+* use the curl http client with the `use_curl_http_client: yes` configuration option in `datadog.conf`
+
+Note: `datadog.conf` is located in `C:\ProgramData\Datadog\datadog.conf`.
+
+In this case, removing `datadog-cert.pem` will not allow the Agent to regain connectivity to Datadog. Instead, take this action:
+
+* Windows Agent v5, `>= 5.12.0`: replace the `datadog-cert.pem` file with the version that is shipped in 5.32.7. Using the Powershell CLI:
+
+```shell
+Invoke-WebRequest -Uri f"https://raw.githubusercontent.com/DataDog/dd-agent/5.32.7/datadog-cert.pem" -OutFile "C:\Program Files\Datadog\Datadog Agent\agent\datadog-cert.pem"
+restart-service -Force datadogagent
+```
+
+* Windows Agent v5, `<= 5.11.x`: set the following option in `datadog.conf` using the `Datadog Agent Manager` program provided by the Agent or by directly editing the `datadog.conf` file:
+  * 64-bit Windows: `ca_certs: C:\Program Files (x86)\Datadog\Datadog Agent\files\ca-certificates.crt`
+  * 32-bit Windows: `ca_certs: C:\Program Files\Datadog\Datadog Agent\files\ca-certificates.crt`
+  After `datadog.conf` has been updated, restart the Datadog Service from the Windows Service Manager.
+
+
 [1]: https://static.datadoghq.com/find_agents_with_connectivity_problems.py
-[2]: /agent/versions/upgrade_to_agent_v7/?tab=linux#from-agent-v5-to-agent-v7
-[3]: /agent/versions/upgrade_to_agent_v6/?tab=linux
-[4]: https://app.datadoghq.com/account/settings#api
-[5]: https://app.datadoghq.eu/account/settings#api
-[6]: https://github.com/DataDog/dd-agent/releases/tag/5.32.7
-[7]: https://s3.amazonaws.com/ddagent-windows-stable/ddagent-cli-latest.msi
-[8]: https://app.datadoghq.com/account/settings?agent_version=5#agent
+[2]: https://app.datadoghq.com/account/settings#api
+[3]: https://s3.amazonaws.com/ddagent-windows-stable/ddagent-cli-latest.msi
```

Comment:
we could use a datadoghq.com link instead of a s3 link: https://windows-agent.datadoghq.com/ddagent-cli-latest.msi

Reply:
we still use the s3 links in our in-app install instructions. Switching to the datadoghq.com link here only could make things confusing for some customers.

Reply:
we'll keep it as is for consistency with other docs